gatsby-config.js

```diff
@@ -43,6 +53,22 @@ module.exports = {
         icon: `src/images/garden-icon.svg`
       }
     },
+    {
+      resolve: `gatsby-plugin-mdx`,
+      options: {
+        gatsbyRemarkPlugins: [
+          'gatsby-remark-smartypants',
+          {
+            resolve: 'gatsby-remark-autolink-headers',
+            options: {
+              className: 'anchor',
+              icon:
+                '<svg class="octicon-link" xmlns="http://www.w3.org/2000/svg" width="16" height="16" focusable="false" viewBox="0 0 16 16"><path fill="none" stroke="currentColor" stroke-linecap="round" d="M3.4 8.4L1.6 6.6C.3 5.2.3 3 1.6 1.6 3 .2 5.2.2 6.5 1.6l3.2 3.2c1.4 1.4 1.4 3.6 0 4.9-.4.4-.8.7-1.3.8m4.2-2.8l1.8 1.8c1.4 1.4 1.4 3.6 0 4.9-1.4 1.4-3.6 1.4-4.9 0l-3.2-3.2c-1.4-1.4-1.4-3.6 0-4.9.4-.4.8-.7 1.3-.8"/></svg>'
+            }
+          }
+        ]
+      }
+    },
```

Comment:
Why would we not be using Garden components? https://www.gatsbyjs.org/docs/mdx/customizing-components/

Reply:
This is a `remark` plugin, not `MDX`. We have to provide a static string template with the `octicon-link` class since we are using the Github flavored markdown styling right now. 

This is a Garden SVG icon, but due to the way the plugin is built I have provided it as a string.